src/main/java/core/basesyntax/CustomFileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CustomFileReaderImpl implements CustomFileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath))) {
+            StringBuilder builder = new StringBuilder();
+            String line = reader.readLine();
+            while (line != null) {
+                lines.add(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + filePath, e);
+        }
+        return lines;
+    }
+}
```

Comment:
The class `CustomFileReaderImpl` should be placed in a specific package that groups similar classes, according to the checklist. Consider moving it to a package like `service` or `service.impl`.